presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -300,28 +317,37 @@ private Page createStatisticsPage(Page aggregationOutput)
     private Page createFragmentsPage()
     {
         Collection<Slice> fragments = getFutureValue(finishFuture);
+        int positionCount = fragments.size() + 1;
```

Comment:
nit: add a comment about why +1

Reply:
Given we added ASCII art to show the page layout, I think this comment is not needed any more :P